README.md

```diff
@@ -93,40 +93,3 @@ bandit_mean{subgraph="QmRDGLp6BHwiH9HAE2NYEE3f7LrKuRqziHBv76trT4etgU"} 3.6531261
 ```
 
 Where "bandit" refers to the reinforcement learning method (Continuum-armed bandit) used to track the market price for each subgraph.
```

Comment:
hmmm, I understand that you moved the installation instructions somewhere...?

Reply:
That's all indexers need to know: https://github.com/semiotic-ai/autoagora/blob/e64e18b929f66d7cce90043526370bf0511f87db/README.md?plain=1#L13-L19

Reply:
I put some of the "dev" docs back https://github.com/semiotic-ai/autoagora/pull/20/commits/9abef99fc96659080bfbfc7b9f18aeee2ca1ebb8